packages/modules/vehicles/bmwbc/api.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+
+# references:
+# https://github.com/bimmerconnected
+# https://bimmer-connected.readthedocs.io/en/latest/
+
+import os
+import json
+import asyncio
+import datetime
+import logging
+import getpass
+from typing import Union
+from modules.common.component_state import CarState
+from modules.common.store import RAMDISK_PATH
+from bimmer_connected.api.client import MyBMWClientConfiguration
+from bimmer_connected.api.authentication import MyBMWAuthentication
+from bimmer_connected.account import MyBMWAccount
+from bimmer_connected.api.regions import Regions
+from bimmer_connected.utils import MyBMWJSONEncoder
+
+
+log = logging.getLogger(__name__)
+
+
+# ------------ Helper functions -------------------------------------
+# initialize store structures when no store is available
+def init_store():
+    store = {}
+    store['refresh_token'] = None
+    store['access_token'] = None
+    store['expires_at'] = None
+    return store
+
+
+# load store from file, if no store file exists initialize store structure
+def load_store():
+    try:
+        tf = open(storeFile, 'r', encoding='utf-8')
+        store = json.load(tf)
+        if 'refresh_token' not in store:
+            store = init_store()
+        tf.close()
```

Comment:
Bitte immer einen Kontexthandler für Dateioperationen nutzen.
```suggestion
        with open(storeFile, 'r', encoding='utf-8') as tf:
            store = json.load(tf)
        if 'refresh_token' not in store:
            store = init_store()
```